component/src/main/java/io/siddhi/extension/io/file/FileSource.java

```diff
@@ -231,9 +231,19 @@
                         name = "header.present",
                         description = "This parameter used to specify a particular text file (eg: CSV) contains a " +
                                 "header line or not. This can either have value true or false. If it's set to " +
-                                "`true` then it indicates a file contains a header line, and it will not process",
+                                "`true` then it indicates a file contains a header line, and it will not process.\n",
                         optional = true, defaultValue = "false",
-                        type = {DataType.BOOL}),
+                        type = {DataType.BOOL}
+                ),
+                @Parameter(
+                        name = "read.only.header",
+                        description = "This parameter used to specify a particular text file (eg: CSV) contains a " +
```

Comment:
@Ajanthy shall we rephrase this as "This parameter used to read only the header or the first line of a particular text file (eg: CSV). This is only applicable if the `mode` is `LINE`. If it's set to false, the full file content will be read line by line.

Reply:
fixed in https://github.com/siddhi-io/siddhi-io-file/pull/91/commits/eda3cf79cdc3e1487776729e63271fb7f5061e77